specify/specify7
# Open interaction dialog when adding prep in ExchangeOut

Fixes #922

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Go to interactions 
- Open Exchange Out 
- Add a prep
- [ ] verify the interaction dialog opens
- [ ] verify that you can add a prep by selecting "By choosing a record set" 
- [ ] verify that you can add a prep by selecting "By entering catalog number" 
- [ ] verify that you can add an unassociated item 
- [ ] test adding a preparation for ExchangeIn
